e3sm_diags/e3sm_diags_driver.py

```diff
@@ -351,7 +351,7 @@ def _run_with_dask(parameters: List[CoreParameter]) -> List[CoreParameter]:
     https://docs.dask.org/en/stable/generated/dask.dataframe.DataFrame.compute.html
     """
     bag = dask.bag.from_sequence(parameters)
-    config = {"scheduler": "processes", "context": "fork"}
+    config = {"scheduler": "processes", "multiprocessing.context": "fork"}
```

Comment:
The freeze_support() issue came back in one of my test cases. When replace with the native keyword `multiprocessing.content`, it solved the problem.